Developers/Unity Project/Spooksville/Assets/Scripts/Items/WeaponManager.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class WeaponManager : MonoBehaviour
+{
+    public static WeaponManager instance;
+    private List<Weapon> weapons = new List<Weapon>();
+
+    public Weapon GetRandomWeapon()
+    {
+        return weapons[Mathf.RoundToInt(Random.Range(0, weapons.Count - 1))];
+    }
+
+    void Start()
+    {
+        if (instance == null)
```

Comment:
you might want to log a warning or something if instance isn't null - doesn't that mean there is more than one player in the scene?